apps/site/util/authorUtils.ts

```diff
@@ -0,0 +1,73 @@
+import authors from '@/authors.json';
```

Comment:
JSON imports should come from our dedicated next.json file, and from there surround the type with a JSDoc type.

Reply:
I missed this, thank you! 🙇